RSSReader/View/FeedViewController.h

```diff
@@ -0,0 +1,22 @@
+//
+//  FeedViewController.h
+//  RSSReader
+//
+//  Created by Владислав Станкевич on 17.11.20.
+//
+
+#import <UIKit/UIKit.h>
+#import "FeedPresenter.h"
+#import "FeedItem.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FeedViewController : UIViewController <UITableViewDataSource, UITableViewDelegate>
+
+@property (nonatomic, retain) UITableView* tableView;
+@property (nonatomic, retain) FeedPresenter* presenter;
+@property (nonatomic, retain) NSMutableArray<FeedItem*>* feedItemArray;
```

Comment:
view controller should be a kinda stupid, don't store models here
instead, provide a method inside the presenter and call it in the view controller to get items